scripts/globals/mobskills/Bombilation.lua

```diff
@@ -0,0 +1,28 @@
+---------------------------------------------
+--  Bombilation
+--
+--  Description: Resets TP of all targets in an area of effect.
+--  Type: Enfeebling
+--  Ignores Shadows
+--  Range: Unknown radial
+---------------------------------------------
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+
+---------------------------------------------
+function onMobSkillCheck(target,mob,skill)
+
+    return 0;
+end;
+
+function onMobWeaponSkill(target, mob, skill)
+   local reset = 0;
```

Comment:
Yeah, you can just push on your end and this PR will be auto-updated. If you want to make it stay as just a single commit, can do a local rebase, mark the second commit as a fixup, and then force-push to remote.